docs/pages/example/add-image-missing-generated.js

```diff
@@ -0,0 +1,11 @@
+/*---
+title: Generate and add a missing icon to the map
+description: Add a missing icon to the map that was generated at runtime.
```

Comment:
Nit: in `the map that was generated at runtime`, "that" appears to reference "map". Maybe something like `Dynamically generate a missing icon at runtime and add it to the map`?

Reply:
yep, that's better. Thanks!